src/auth/auth.ts

```diff
@@ -520,8 +520,9 @@ export class BaseAuth<T extends AbstractAuthRequestHandler> implements BaseAuthI
   public verifySessionCookie(
     sessionCookie: string, checkRevoked = false): Promise<DecodedIdToken> {
     const isEmulator = useEmulator();
-    return this.sessionCookieVerifier.verifyJWT(sessionCookie, isEmulator)
+    return this.sessionCookieVerifier.verifyJWT<DecodedIdToken>(sessionCookie, isEmulator)
       .then((decodedIdToken: DecodedIdToken) => {
+        decodedIdToken.uid = decodedIdToken.sub;
```

Comment:
Moved the code to set the `uid` alias here. Another way is to add helper functions, `verifySessionCookie()` and `verifyIdToken()`, to `auth/token-verifier`. Let me know your thoughts.